doc/source/working_with_kiwi/legacy_kiwi.rst

```diff
@@ -39,25 +40,25 @@ In order to address all of these the questions came up:
   `How to change/drop features without making anybody unhappy?`
 
 As there is no good way to achieve this in the former code base the
-decision was made to start a rewrite of KIWI with a maintained and
+decision was made to start a rewrite of {kiwi-legacy} with a maintained and
 stable version in the background.
 
 Users will be able to use both versions in parallel. In addition, the
-next generation KIWI will be fully compatible with the current format of
+{kiwi-product} will be fully compatible with the current format of
 the appliance description. This means, users can build an appliance from
-the same appliance description with the legacy and the next generation
-KIWI, if the distribution and all configured features are supported by
-the used KIWI version.
+the same appliance description with the {kiwi-legacy} and the 
+{kiwi-product}, if the distribution and all configured features are
+supported by the used version.
 
-This provides an opportunity for users to test the next generation KIWI
+This provides an opportunity for users to test the {kiwi-product}
 with their appliance descriptions without risk. If it builds and works
-as expected, I recommend to switch to the next generation KIWI. If not,
+as expected, I recommend to switch to the {kiwi-product}. If not,
 please open an issue on https://github.com/OSInside/kiwi.
```

Comment:
In my opinion, `{kiwi-product}` should be used to *introduce* the term. After you have used it, you just use `{kiwi}`.

Reply:
This applies to all other chapters but not the explanations we use in this legacy chapter. I suggest to build it and read the result. For me that looked ok

Reply:
sorry I got it wrong, yes I agree will update the text